engine/dlib/src/wscript

```diff
@@ -41,6 +41,8 @@ def build(bld):
         platform_source_dirs.append('dlib/ps4/**/*.cpp')
 
     extra_defines = ['ZLIB_CONST']
+    extra_defines +=['MINIZ_NO_TIME', 'MINIZ_NO_DEFLATE_APIS']
+    extra_defines +=['MINIZ_USE_ANSI', 'ZIP_NO_DEFLATE', 'ZIP_NO_DISC_IO']   # defold defines
```

Comment:
I introduced these, in order to get it working.
I'll create another PR upstream for at least zip.c